### DISCUSSION TITLE
Can't Submit Developer Pack Application

### DISCUSSION THREAD

**Discussion by Astrobot-me** [community/community#132027] [GitHub Education]
### Hi everyone,

I upload all of my details i.e. Academic proof and a Picture of a Notice that the Semester break is going on that's why I'm away from campus as proof and When I click Process and submit my application, it finishes loading bar to 100% and then refreshes the page. There is no other response after doing that and it just refreshes the page and asks again me to upload without actually suggesting to me what I did do wrong? How can I submit it?

Any suggestions

**Comment by naveenalla3000** [comment]
do it again bro.

**Reply by Astrobot-me** [reply]
Tried so many time as of now .. same thing happening again n again : (

**Comment by naveenalla3000** [comment]
make sure that institution mail is linked with GitHub and clear photo of your Id from institution(academics years mention)

mark this comment as answer if it helpful.

**Reply by Astrobot-me** [reply]
The institution is Mail is Linked with my account and verified as well, secondly as my valid up to date in mentioned on the backside of the Identity card. I made a pdf file of the identity card of the front side and back sides.. and then took a photo from my phone of the identity card displayed on the laptop screen . 

Not Working

**Reply by naveenalla3000** [reply]
Do upload as photo from laptop

**Reply by Astrobot-me** [reply]
> Do upload as photo from laptop

It only allows you to take photos from camera and doesn't provide any upload options

**Comment by TheCukitoDev** [comment]
I have the same problem. I oppened a ticket and they clossed me inmediatly with a response of other type.
![image](https://github.com/user-attachments/assets/3a572115-f448-4ba0-9de4-b2d86b94978f)

**Reply by Astrobot-me** [reply]
They need to fix this problem.. of refreshing and not submitting or may be something we need to fix on our part

**Reply by TheCukitoDev** [reply]
i didn't lose any of my benefits as a student. I has copilot, gh pro, ... They didn't solved the problem but also didn't removed my benefits.

**Reply by TheCukitoDev** [reply]
Hello, I aalready could send an aplication.

**Reply by Astrobot-me** [reply]
> Hello, I aalready could send an aplication. 

Is the problem resolved? Are you able to submit your application successfully without that refresh issue?

**Reply by TheCukitoDev** [reply]
Yes. I did it the normal way and it worked. A problem were that i've got blocked the location in the pc and the browser can't give any location also if u give the permission to the web. You have to make sure your browser have pc permission to the location. The i think it should go well. After that i went rejected 2 or 3 times because of some things but i almost could send the requests. I hope this will help you.
